modules/postfix/files/aliases

```diff
@@ -16,22 +16,23 @@ void-recipient: /dev/null
 # root spam
 root: /dev/null
 
-# operations aliases
-ops: operations
+# redirect for old names
+operations: sre
+staff: sysadmins
```

Comment:
I doint really like this but if John supports it then I won’t block this.

How about tech@? Nicer name.

Reply:
Tech@ sounds great and I did say I’m open to nicer names!!